### PR TITLE
[IMP] l10n_cl_electronic_invoicing: Documentation

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by max3903** [comment]
Ping @nelsonramirezs

File: l10n_cl_electronic_invoicing/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+Journals
+--------
+
+* Go to Accounting > Configuration > Journals
+* Select the Customer Invoices or Vendor Bills journal
+* Enter the documents that can be generated
+
+|step5|
+
```
Comment (bodedra) [inline l10n_cl_electronic_invoicing/readme/CONFIGURE.rst:97]: @max3903 Step 5 is empty. :thinking:

File: l10n_cl_electronic_invoicing/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+* Go to Settings > User & Companies > Companies
+* Edit the company
+* Set the "DTE Service Provider" to "www.sii.cl"
+
+|step12|
+
+* Go to https://maullin.sii.cl/cvc/dte/pe_condiciones.html
+* Update your data to set the email address configured to receive vendor bills
+
+|step13|
```
Comment (bodedra) [inline l10n_cl_electronic_invoicing/readme/CONFIGURE.rst:151]: Same here.

File: l10n_cl_electronic_invoicing/readme/CONFIGURE.rst (hunk 1 above)
Comment (max3903) [reply l10n_cl_electronic_invoicing/readme/CONFIGURE.rst:97]: @bodedra What do you mean by empty? The image is declared with the others. Once the README.rst will be generated at the root of the repo, image paths should work.

Comment (bodedra) [reply l10n_cl_electronic_invoicing/readme/CONFIGURE.rst:97]: ah. I got it. I was expected content like have in others.

My comment is invalid. :smile:

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Comment by nelsonramirezs** [comment]
LGTM2!

**Review by nelsonramirezs — APPROVED** [review]

**Review by nelsonramirezs — APPROVED** [review]

**Comment by max3903** [comment]
@flectra-chile Can you review this pull request please?

**Pull request merged**